examples/nrunner/recipes/runnables/sysinfo.json

```diff
@@ -0,0 +1 @@
+{"kind": "sysinfo", "uri": "pre", "kwargs": {"sysinfo_files": {"commands": ["uptime", "dmidecode"], "files": ["/proc/version", "/proc/meminfo"]}}}
```

Comment:
Running this recipe:

```
$ avocado-runner-sysinfo runnable-run-recipe examples/nrunner/recipes/runnables/sysinfo.json 
```

Results in:

```
{'status': 'started', 'time': 78926.805707241}
{'result': 'pass', 'status': 'finished', 'time': 78926.808248625}
```

I assume the runnable configuration is missing here (say, coming from `settings.as_dict()`)... which is not a problem with this PR, but it shows that recipes, supposed to be easy to be executed again, are deviating from that purpose.  

IMO we need to revisit what we require from runnables, so that we don't lose that feature.

Reply:
Thanks for catching this, I was resolving @beraldoleal's comments, and created this bug. I will fix it.